package.json

```diff
@@ -179,6 +180,7 @@
     "netlify-cli": "^10.0.0",
     "postcss": "^8.4.12",
     "readable-stream": "^4.1.0",
+    "serverless": "^3.22.0",
```

Comment:
This is not actually a new package, I was already using this for local development, but I had it as a global dependency.

The reason why I am not using `netlify-cli` is because it's HMR doesn't work correctly.
It has a lot of bugs, so working with it was really a pain.
That's why I moved to `serverless` for local-dev. It's much more stable.

Reply:
ah cool thanks. i need to look into Netlify CLI inner workings cause I'd like us to do a similar service for bitcoin applications (an SDK).